files/livefyre/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "LiveFyre"
+github = "https://github.com/itseasy21/jsdelivr"
```

Comment:
It doesnt look like its an open source project. We dont just host js files. It needs to have a project page and a license.